courses/free-courses-en.md

```diff
@@ -330,6 +330,7 @@
 * [CryptoZombies.io](https://cryptozombies.io) - CleverFlare, Loom Network
 * [Ethereum Developer Bootcamp](https://www.alchemy.com/university/courses/ethereum) - Alchemy University
 * [Learn Blockchain](https://www.youtube.com/playlist?list=PLlp912GlUiC1xPnwVmKgIiuMERetMwre9) - Roomyan
+* [Learn Blockchain & Solidity Development Full Course 2024](https://youtu.be/jcgfQEbptdo?feature=shared) - Dapp University
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
NO SHORTENED LINKS